math/libfaiss/Portfile

```diff
@@ -60,3 +62,92 @@ variant openblas description {Use OpenBLAS instead of Apple's Accelerate framewo
 }
 
 default_variants +openblas
+
+# create Python subports
+set python_versions {37 38}
+foreach v ${python_versions} {
+    subport py${v}-${github.project} {
+        PortGroup   python 1.0
+        python.default_version  ${v}
+    }
+}
+
+# Python bindings for supported Python versions
+if {[string match "py*" ${subport}]} {
+    github.setup        ${github.author} ${github.project} ${github.version} ${github.tag_prefix}
+
+    categories-append \
+                    math
+    description     Python ${python.branch} bindings for ${description}
+    long_description \
+                    This package provides Python ${python.branch} bindings, \
+                    modules, and scripts.
+
+    depends_build-append \
+                    port:py${python.version}-setuptools \
+                    port:swig \
+                    port:swig-python
+
+    depends_lib-append \
+                    port:${name} \
+                    port:py${python.version}-numpy
+
+    depends_test-append \
+                    port:py${python.version}-pytest
+
+    # Build instructions:
+    # https://github.com/facebookresearch/faiss/blob/master/INSTALL.md#compile-from-source
+
+    use_configure   yes
+
+    configure.args-replace \
+                    -DFAISS_ENABLE_PYTHON:BOOL=OFF \
+                    -DFAISS_ENABLE_PYTHON:BOOL=ON
+    configure.args-append \
+                    -DPython3_EXECUTABLE=${python.bin} \
+                    -DSWIG_EXECUTABLE=${prefix}/bin/swig
+    configure.pre_args-delete \
+                    --prefix=${prefix}
+    configure.pre_args-replace \
+                    {-DCMAKE_C_COMPILER="$CC"} \
+                    -DCMAKE_C_COMPILER=${configure.cc}
+    configure.pre_args-replace \
+                    {-DCMAKE_CXX_COMPILER="$CXX"} \
+                    -DCMAKE_CXX_COMPILER=${configure.cxx}
```

Comment:
Why is this needed? The existing lines are supposed to set the CMake C and C++ compilers to whatever the CC and CXX environment variables are. Why is that not sufficient?

Reply:
Thanks not needed.